### PR TITLE
Displaying currentTime and correct icon when song ends

### PR DESCRIPTION
**Pull request by janeslee**

### CODE REVIEW

**Comment by janeslee** [comment]
- Implemented functionality to skip to different segments of the song via clicking the wave form

File: client/src/components/MusicPlayer.jsx [hunk 1]
```diff
@@ -1,51 +1,68 @@
...
 import styles from '../styles/MusicPlayer.css';
 import axios from 'axios';
 import TimeAgo from 'react-timeago';
 
-
-const imagePaths = {
-  pauseButton: 'https://s3-us-west-1.amazonaws.com/democrituscloud/pause.png',
-  playButton: 'https://s3-us-west-1.amazonaws.com/democrituscloud/play.png'
-}
+var audio;
```
Comment (therobinkim) [inline client/src/components/MusicPlayer.jsx:8]: just curious, what were the tradeoffs for having `audio` as a global var instead as part of state?

File: client/src/components/MusicPlayer.jsx [hunk 2]
```diff
@@ -55,29 +72,59 @@ class MusicPlayer extends React.Component {
...
+
+    // controls song play and pause
     this.setState({
-      play: !this.state.play
+      play: !this.state.play,
     });
+
     if (this.state.play === true) {
-      this.audio.pause();
+      audio.play();
```
Comment (therobinkim) [inline client/src/components/MusicPlayer.jsx:101]: for the line above, you can just say `if (this.state.play) {`

File: client/src/components/MusicPlayer.jsx [hunk 3]
```diff
@@ -55,29 +72,59 @@ class MusicPlayer extends React.Component {
...
+      this.setState({
+        playerIcon: 'https://s3-us-west-1.amazonaws.com/democrituscloud/play.png'
+      });
+      // clearInterval(this.timer);
     }
   }
 
+  skipToSegment(position) {
+    // jump to a new segment in the song based on the position in the wave form
+    audio.currentTime = position * (this.state.duration / 241);
```
Comment (therobinkim) [inline client/src/components/MusicPlayer.jsx:116]: interesting! what is the significance of `241`?

**Pull request merged**